tests/src/test/resources/templates/build.gradle.mustache

```diff
@@ -0,0 +1,103 @@
+apply plugin: 'idea'
```

Comment:
I get this file via `./gradlew :tests:generateSwaggerCodeJava` and replace `jcenter()` to `mavenCentral()`